daisy_integration_tests/scripts/post_translate_test.ps1

```diff
@@ -45,16 +71,29 @@ function Check-Activation  {
   }
 }
 
+function Check-SkipActivation {
+  if (!((Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SoftwareProtectionPlatform').KeyManagementServiceName -eq $null)) {
+    throw 'Expected: Activation Skipped, Found: Activated License'
+  }
+}
+
 try {
+  $script:byol = Get-MetadataValue -key 'byol'
```

Comment:
Set the -default flag here to avoid the error in lookup, also the 'script' scoping is unneeded as it's just used in this block and not elsewhere: 
$byol = Get-MetadataValue -key 'byol' -default 'false'

Reply:
Updated.